snapshotTestContent/DeletedTables_0.43/src_snapshots/0.59.3000/snapshot_461_0.json

```diff
@@ -86,88 +86,16 @@
-                                              "encoding": "utf-8"
-                                            }
-                                          }
-                                        ]
-                                      }
-                                    },
-                                    {
-                                      "mode": "100644",
-                                      "path": "handleTable",
-                                      "type": "Blob",
-                                      "value": {
-                                        "contents": "[2,0,5,0,0]",
-                                        "encoding": "utf-8"
-                                      }
-                                    }
-                                  ]
-                                }
-                              },
-                              {
-                                "mode": "040000",
-                                "path": "cols",
-                                "type": "Tree",
-                                "value": {
-                                  "entries": [
-                                    {
-                                      "mode": "040000",
-                                      "path": "segments",
-                                      "type": "Tree",
-                                      "value": {
-                                        "entries": [
-                                          {
-                                            "mode": "100644",
-                                            "path": "header",
-                                            "type": "Blob",
-                                            "value": {
-                                              "contents": "{\"version\":\"1\",\"segmentCount\":1,\"length\":2,\"segments\":[[2,1]],\"startIndex\":0,\"headerMetadata\":{\"minSequenceNumber\":461,\"sequenceNumber\":461,\"orderedChunkMetadata\":[{\"id\":\"header\"}],\"totalLength\":2,\"totalSegmentCount\":1}}",
-                                              "encoding": "utf-8"
-                                            }
-                                          }
-                                        ]
-                                      }
-                                    },
-                                    {
-                                      "mode": "100644",
-                                      "path": "handleTable",
-                                      "type": "Blob",
-                                      "value": {
-                                        "contents": "[5,0,0,0,6,4]",
-                                        "encoding": "utf-8"
-                                      }
-                                    }
-                                  ]
-                                }
-                              },
```

Comment:
Why is this getting deleted?

Reply:
Looks like we should automatically sort everything in order.

Reply:
We actually do that in snapshot normalizer when we normalize the snapshots. However, when we write the snapshots, we write whatever the runtime returns without filtering anything. And that is what causes this to change from time to time.
The idea is that we preserve the summary that runtime returns so that later we can choose to start processing properties that the snapshot normalizer removes / updates.
However, we can sort the entries in the summary so we don't keep updating it.